bitsail-dist/src/main/resources/bin/bitsail

```diff
@@ -35,6 +35,11 @@ BITSAIL_CLIENT_CLASSPATH=${BITSAIL_HOME}/libs/clients
 if [ -n "$HADOOP_HOME" ]; then
   export HADOOP_CLASSPATH=$($HADOOP_HOME/bin/hadoop classpath)
 fi
+
+if [ -z "$BITSAIL_JAVA_OPTS" ]; then
+  export BITSAIL_JAVA_OPTS="-Xmx128M"
```

Comment:
can we get the java opts from the environment variable? maybe we will add other opts later

Reply:
> can we get the java opts from the environment variable? maybe we will add other opts later

The current implementation checks whether the environment variable `BITSAIL_JAVA_OPTS` exists. If it exists, it will use the existing environment variable; otherwise, it will provide a default value to prevent excessive memory usage.

Reply:
got it.